flink_jobs_v2/batch_status/.gitignore

```diff
@@ -5,4 +5,4 @@
 .classpath
 /nbproject
 nbactions.xml
-
+/src/main/resources/test/results.txt
```

Comment:
Why is that in gitignore? seems a bit strange

Reply:
Removed. as it was generated each time the test was running it appeared as changed and by error it was uploaded. 
As the datasink now is fixed to not write a file it is not needed